### DISCUSSION TITLE
lamp-cloud-pro-datasource-column 的资源维护中的"字段"实现

### DISCUSSION THREAD

**Discussion by teng-jy** [zuihou/zuihou-admin-cloud#240] [02.提问]
![image](https://user-images.githubusercontent.com/62694039/223329715-dc2e29f5-2530-4b90-8007-bfc13116d4f8.png)
这里的字段是区分不同人进来看到的字段是不同的吗？ 是否已经实现了。如果实现了能简单描述一下这里是如何使用。如果没有实现能说下如何实现或者一种思路吗

**Comment by zuihou** [comment]
在 有表格的 页面，控制不同的人看不同字段

![image](https://user-images.githubusercontent.com/8500410/223335770-cba17ac6-cd65-4dc7-a066-1bda676fc878.png)

**Reply by teng-jy** [reply]
可以解答一下 key 与 value 是什么意思嘛？ 如您截的图 key是name ,value 是test ?  我配置进去然后我给这个人这个权限就只有这个人能看到这个”name“ 字段了嘛？

**Reply by zuihou** [reply]
auth:  'test'  中的test 配置成“资源维护”中的“编码”

**Reply by teng-jy** [reply]
明白了 感谢
